src/index.ts

```diff
@@ -138,6 +138,7 @@ app.post("/", async (req, res, next) => {
       const backrunResult = await mevshare.sendBundle(bundleParams);
 
       Logger.debug(req.transactionId, "Forwarded a bundle to MEV-Share", { bundleParams });
+      Logger.info(req.transactionId, "Bundle accepted by MEV-Share", { bundleHash: backrunResult.bundleHash });
```

Comment:
for now this is fine but in the future it would be nice to expand this some. something like knowing how many elements are in the bundle and including a link to the unlock latest value transaction, for example.